### ISSUE TITLE
Spanish model throws key error

### ISSUE THREAD

**Issue by oliverdain** [explosion/spaCy#1151]
Having an issue with the Spanish model:

```
def test_spacy():
    nlp = spacy.load('es_core_web_md')
    doc = nlp("""Hay alguno cosas que me hacemos una persona la única. La
              primera característica es que soy muy talentoso en los idiomas
              extraños.  Para mí, aprender un idioma extranjera no es un tarea
              muy difícil. La otra característica que me único hace es que sea
              muy resuelto. Cuando yo era niño, y jugaba deportes, detestaba
              perder un juego. Al final, soy un hombre que tiene talentas en
              los idiomas extrajeros, y es muy resuelto.""")
    assert doc.is_parsed
```

results in:

```
build/virtualenv/lib/python3.5/site-packages/spacy/language.py:320: in __call__
    proc(doc)
spacy/tagger.pyx:212: in spacy.tagger.Tagger.__call__ (spacy/tagger.cpp:6245)
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   KeyError: 690

spacy/morphology.pyx:76: KeyError
```

Note that I posted this in https://github.com/explosion/spaCy/issues/1052 but I think maybe since that issue was, at one point, marked as resolved nobody is looking at it. If that's incorrect I apologize for the duplicate.

**Comment by oliverdain** [comment]
This is with spacy 1.8 and the model here: https://github.com/explosion/spacy-models/releases/download/es_core_web_md-1.0.0/es_core_web_md-1.0.0.tar.gz

**Comment by oliverdain** [comment]
Appears to be fixed with 1.8.2. Sorry, didn't realize I didn't have the latest version.
